virtualization/windowscontainers/docker/configure_docker_daemon.md

```diff
@@ -31,19 +31,19 @@ Download the Docker Engine
 
 The latest version may always be found at https://master.dockerproject.org . This sample uses the latest from the v1.13-development branch. 
 
-```none
+```powershell
 Invoke-WebRequest "https://download.docker.com/components/engine/windows-server/cs-1.12/docker.zip" -OutFile "$env:TEMP\docker.zip" -UseBasicParsing
```

Comment:
I've had v1.13-dev on my Windows 10, but needed to switch to cs-1.12.2 after GA version of container images for Nano Server where uploaded to Docker Hub. What I have on Windows 10 now is the following:
```
PS C:\> docker version
Client:
 Version:      1.12.2-cs2-ws-beta
 API version:  1.25
 Go version:   go1.7.1
 Git commit:   050b611
 Built:        Tue Oct 11 02:35:40 2016
 OS/Arch:      windows/amd64

Server:
 Version:      1.12.2-cs2-ws-beta
 API version:  1.25
 Go version:   go1.7.1
 Git commit:   050b611
 Built:        Tue Oct 11 02:35:40 2016
 OS/Arch:      windows/amd64
```